content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -54,13 +54,13 @@ The CPU consumption scales with the following factors:
 - The rate of configuration changes.
 - The number of proxies connecting to Istiod.
 
-however this part is inherently horizontally scalable.
+However, this part is inherently horizontally scalable.
 
-When [namespace isolation](/docs/reference/config/networking/sidecar/) is enabled,
```

Comment:
This change is not strictly related ,but these numbers are basically made up junk from Istio 1.0. Its misleading to users to claim such a simple mapping of services => CPU.